server/README.md

```diff
@@ -70,3 +68,37 @@ The `server` is the back-end to TheHaloMod-SPA application. It provides the API,
 The entrypoint of the server is [`server/halomod_app/__init__.py`](halomod_app/__init__.py) which builds the Flask application and adds routes to it. This file is used as the main location for the top-level server functionality. The [`server/halomod_app/utils.py`](halomod_app/utils.py) houses the extra functionality or constants that have been abstracted out of the main server logic. 
 
 Besides the things mentioned here, documentation exists in each file to provide guidance on what each does. 
+
+
+## Manual setup, server start, testing and linting
+Follow the steps below if you are unable to use [run.sh](./run.sh). Use run.sh as a reference to this section.
+
+Set up virtual env
+```sh
+python3 -m venv env
+. env/bin/activate
+```
+Dependency installation
+```sh
+pip3 install -r requirements.txt;;
+```
+Starting the server
+```sh
+export FLASK_APP=halomod_app
+export FLASK_ENV=development
+flask run;;
+```
+Testing
+```sh
+python3 -m pytest;;
+```
+Linting
+```sh
+autopep8 -a -r --in-place halomod_app # fixes autofixable lint errors
+flake8 tests halomod_app;;
+```
```

Comment:
It looks like the new sections need to be added to the table of contents for this readme.